src/index.html

```diff
@@ -24,9 +24,16 @@ <h2>Today's To Do</h2>
         </header>
 
         <div id="todos-list">
+            <!-- <div class="todo" id="0">
+                <i class="bi bi-circle"></i>
+                <i class="bi bi-check-circle-fill"></i>
+                <p class="checked">Go get milk.</p>
+                <i class="bi bi-pencil-square"></i>
+                <i class="bi bi-trash"></i>
+            </div> -->
```

Comment:
- [ ] [OPTIONAL] Let's remove these unused codes in order for us to have a clean code base.